probe/awsecs/reporter.go

```diff
@@ -153,7 +153,12 @@ func (r Reporter) Tag(rpt report.Report) (report.Report, error) {
 				ServiceDesiredCount:   fmt.Sprintf("%d", service.DesiredCount),
 				ServiceRunningCount:   fmt.Sprintf("%d", service.RunningCount),
 				report.ControlProbeID: r.probeID,
-			}).WithLatestActiveControls(ScaleUp, ScaleDown))
+			}).WithLatestControls(map[string]report.NodeControlData{
+				ScaleUp: {Dead: false},
+				// We've decided for now to disable ScaleDown when only 1 task is desired,
+				// since scaling down to 0 would cause the service to disappear (#2085)
+				ScaleDown: {Dead: service.DesiredCount <= 1},
```

Comment:
You should also check this in the control itself, and not allow rescaling below 1 in case the UI gets out of sync